src/createAnimatedComponent.tsx

```diff
@@ -115,11 +120,18 @@ const has = <K extends string>(
   return false;
 };
 
-function isInlineStyleTransform(transform: any): boolean {
-  if (!transform) {
+function isInlineStyleTransform(
+  transform: unknown
+): transform is TransformsStyle['transform'] {
```

Comment:
What's the point of having `transform is TransformsStyle['transform']` here? The function may contain valid a transform but without inline styles and in such case the argument won't be treated as `TransformsStyle['transform']` because the function returns `false`, am I correct?

Reply:
Almost - it will return true for both inline transform and transform returned from useAnimatedStyle. However I will be working on a new feature for transformations so it should be safe to close this PR and move all this fix to the new feature

Reply:
We have inline styles and styles returned from a hook "useAnimatedStyles". The returned styles are wrapped inside `AnimatedStyleResult`, so you need to extract them from this object first to make `isInlineStyleTransform` return true.